### DISCUSSION TITLE
How about training a DETR model on the VCOCO dataset

### DISCUSSION THREAD

**Discussion by ddwhzh** [fredzzhang/upt#30] [Q&A] [answered]
Thank you for your excellent work. 
you have provided a tutorial on training DETR models on the HICO-DET dataset, could you tell us how you trained the DETR  on the VCOCO dataset?

**Comment by fredzzhang** [comment, marked as answer]
Hi @ddwhzh,

The code for fine-tuning on V-COCO can be found in the same repository, under the branch `vcoco`, you do need to set up a softlink pointing to the V-COCO dataset though.

Cheers,
Fred.

**Comment by ddwhzh** [comment]
Sorry for the possible misunderstanding due to the expression, I would like to know how to pre-train a detr pre-train model on vcoco instead of training upt with a pre-trained model.

**Comment by fredzzhang** [comment]
The `vcoco` branch is precisely for that purpose. If you look at the history of the branch, you'll find that commit [`906fb89`](https://github.com/fredzzhang/hicodet/commit/906fb8936fd11bc0e9b9fabc3ae7ff46ac2bd117#diff-af200873a2148e22f4cbc3d7c65bd9b173b5591b17d460097918348837147176) added support for fine-tuning the object detector on V-COCO.

**Comment by ddwhzh** [comment]
THX, understood.
